pkg/btree/bptree_test.go

```diff
@@ -221,10 +232,15 @@ func TestBPTree_SequentialInsertionTest(t *testing.T) {
 	for i := 0; i < 256; i++ {
 		buf := make([]byte, 8)
 		binary.BigEndian.PutUint64(buf, uint64(i))
-		if err := tree.Insert(ReferencedValue{Value: buf}, MemoryPointer{Offset: uint64(i)}); err != nil {
+		if err := tree.Insert(ReferencedValue{Value: buf}, MemoryPointer{Offset: uint64(i), Length: uint32(len(buf))}); err != nil {
 			t.Fatal(err)
 		}
 	}
+
+	if err := b.WriteToDisk("bptree_sequential.bin"); err != nil {
```

Comment:
It would be useful to make this a separate entrypoint. You can make a `test/sequential/main.go` or something to generate the binary file. That way the entire test suite doesn't have to be run to generate the file and the tests stay side-effect-free.